### DISCUSSION TITLE
mixed column filters: date & tag

### DISCUSSION THREAD

**Discussion by dictionarymouse** [roovo/obsidian-card-board#249] [Ideas]
It'd be nifty to have the ability to combine **date _and_ tag filters** for different columns.
For example, I'd like to setup a board something like this: 

"Morning" filter = _Today_ ➕  _#am_ **OR** _undated_ ➕  _#am_
"Urgent" filter = _Before Today_ (overdue) **OR** _#frog_ 
"Today" filter = _Today_  ➖ _#am_ ➖ _#pm_
 "Unscheduled" filter = _undated_  ➖  _#am_ ➖ _#pm_
 "Evening" filter =  _Today_ ➕ _#pm_ **OR** _undated_ & _#pm_
"Tomorrow" 
"This week" (any date this week)
"Later" (future dates)

**Comment by martex33** [comment]
Thanks for this really helpful plug-in. I also would benefit from a combination of filters. 
A second level filter could be easily done and would already help a lot. More could be useful as well.
Tag filters should have a polarity (including/excluding).
Example: Currently I would need a dated or undated column but without the tag "#wait". For tasks with "#wait" I reserved a extra column.
Maybe allowing additional tag filters would already be enough to boost the flexibility.
Kind regards,
Martin
